src/Components/Web.Extensions/src/InputFile/BrowserFile.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    internal class BrowserFile : IBrowserFile
```

Comment:
Make `BrowserFile` `public sealed`, deserialize using some internal type and copy properties.